### PR TITLE
[YUNIKORN-638] Make placeholder image configurable

### PR DESCRIPTION
**Pull request by pbacsko**

### CODE REVIEW

**Review by craigcondit — APPROVED** [review]
+1, pending changes to shim.

File: helm-charts/yunikorn/templates/deployment.yaml [hunk 1]
```diff
@@ -83,6 +83,10 @@ spec:
               value: "{{ .Values.UserLabelKey }}"
             - name: OPERATOR_PLUGINS
               value: "{{ .Values.operatorPlugins }}"
+            {{- if .Values.placeholderImage }}
```
Comment (yuchaoran2011) [inline helm-charts/yunikorn/templates/deployment.yaml:86]: Can we remove the `if` block and instead add a default image to `values.yaml`?

Comment (craigcondit) [reply helm-charts/yunikorn/templates/deployment.yaml:86]: I agree, this is cleaner.

**Review by craigcondit — CHANGES_REQUESTED** [review]
Let's refactor as Chaoran suggested.

**Review by yuchaoran2011 — APPROVED** [review]

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Review by wilfred-s — APPROVED** [review]

**Pull request closed without merging**